### ISSUE TITLE
STILL an issue with short grass

### ISSUE THREAD

**Issue by tsingyayin** [AdvancedPlugins/Seasons#192]
### Describe the bug

Thank you to the author for actively updating this plugin.

My plugin has been updated to 1.6.3 (the latest version on Spigot at this moment), and it seems that improvements regarding snow generation were mentioned in a previous update, while also closing my old issue #139 . This issue aims to point out that after running the plugin on the server for a long time, the short grass will eventually fill the entire terrain.

However, the problem is that although this issue has been closed, I believe it still exists. Let me rephrase it, I think (perhaps subjectively) that this plugin should follow this logic when processing `short grass` and `snow`:

If `generateCustomGrass` in `blocks.yml` is set to `false`, then **only snow should be generated** in the air block. In spring, **only melt the snow generated by this plugin without changing any other blocks**.

However, currently, when the value is configured as above, spring **still** generates short grass, ultimately leading to the entire server being covered in short grass.

I did observe that when the value was set to `true`, some short grass was removed. It has been confirmed that in this situation, there will be no phenomenon of short grass flooding on the server for a considerable period of time. It seems that setting it to `true` can solve the problem I mentioned earlier —— BUT, the problem is that many players also use these small things on the land to decorate their houses, and this configuration will destroy their backyard.

For my players, this plugin should provide the function of allowing them to live in different vanilla biomes according to the season, rather than having them busy mowing their backyard in spring. 

Therefore, I still hope (and sincerely implore) the author to continue considering whether further improvements are needed regarding the relevant issues. That is:

When the value is set to 'false', the plugin only adds and removes snow, and will not modify any other blocks. This logic is also written in the comments in the configuration file, and it should be presented correctly.

```yaml
worlds:
  world:
    # Setting this to false will stop plugin from removing grass and generating snow only in empty AIR blocks.
    # It will also make new grass to not generate when snow is clearing - old grass will stay.
    generateCustomGrass: false
```

### How to reproduce

1. blocks.yml:world.[worldname].generateCustomGrass:false
2. winter->spring
3. snow->short grass
4. repeat the above two steps
5. flood

### Screenshots / Videos

![Screenshot 2024-12-22 220250](https://github.com/user-attachments/assets/d72312ec-cf9d-4c6b-b146-387ad2a27070)

**Comment by GC-spigot** [comment]
I've reduced the density of grass, this should be better now.
